examples/routing_service/routing_service_msql_adapter/cpp/RsStreamReader.hpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * (c) 2020 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ *
+ * RTI grants Licensee a license to use, modify, compile, and create derivative
+ * works of the Software.  Licensee has the right to distribute object form
+ * only for use with RTI products.  The Software is provided "as is", with no
+ * warranty of any type, including any warranty for fitness for any purpose.
+ * RTI is under no obligation to maintain or support the Software.  RTI shall
+ * not be liable for any incidental or consequential damages arising out of the
+ * use or inability to use the software.
+ */
+
+#ifndef RS_STREAM_READER_
+#define RS_STREAM_READER_
+
+#include "ConnectorMsqlDb.hpp"
+#include "RsConnection.hpp"
+#include <chrono>
+#include <dds/core/xtypes/DynamicData.hpp>
+#include <dds/sub/SampleInfo.hpp>
+#include <mutex>
+#include <rti/routing/adapter/StreamReader.hpp>
+#include <thread>
+
+class RsStreamReader : public rti::routing::adapter::DynamicDataStreamReader {
+public:
+    RsStreamReader(
+            RsConnection *connection,
+            const rti::routing::StreamInfo &info,
+            const rti::routing::PropertySet &properties,
+            rti::routing::adapter::StreamReaderListener *listener);
+
+    virtual ~RsStreamReader();
+
+    void
```

Comment:
Fix format

Reply:
It looks like these errors are being reported by Travis CI:

https://travis-ci.org/rticommunity/rticonnextdds-examples/builds/651715468?utm_source=github_status&utm_medium=notification

Reply:
Done. Will commit all changes at once.